lib/tasks/test-sass-compilation.js

```diff
@@ -5,42 +5,72 @@ const buildSass = require('./build-sass');
 const Listr = require('listr');
 const ListrRenderer = require('../helpers/listr-renderer');
 const isCI = require('is-ci');
+const { camelCase } = require('lodash');
+const ftSass = require('@financial-times/sass');
+const fs = require('fs');
+const execa = require('execa');
 
 async function compilationTest(cwd, { silent, brand } = {
 	silent: false,
 	brand: false
 }) {
-	const silentSass = require('../plugins/silent-sass');
 	const mainSassFilePath = await files.getMainSassPath(cwd);
 	if (!mainSassFilePath) {
 		return;
 	}
-	const sassFiles = await files.getSassFilesList(cwd);
-	const supportsSilent = await files.sassSupportsSilent(sassFiles, cwd);
-	if (!supportsSilent) {
-		return;
-	}
 
 	const name = await files.getComponentName();
-	const sassVar = '$' + name + '-is-silent: ' + String(Boolean(silent)) + ';';
-
-	const sassConfig = {
-		sassPrefix: sassVar,
-		sassIncludePaths: [cwd],
-		brand: brand,
-		buildFolder: 'disabled',
-		cwd: cwd,
-		sourcemaps: false,
-		outputStyle: 'compressed' // to remove sass multiline/loud comments from the css output
-	};
+	const primaryMixinName = camelCase(name);
+	const primaryMixinErrorCode = `OBT_NO_PRIMARY-MIXIN-${primaryMixinName}`;
+	const sassContent = fs.readFileSync(mainSassFilePath, 'utf8');
+
+	const sassData = `
+		$system-code: "origami-build-tools";
+		${brand ? `$o-brand: ${brand};` : ''}
 
+		${sassContent}
+
+		${!silent ? `
+			@if not mixin-exists('${primaryMixinName}') {
+				@error '${primaryMixinErrorCode}';
+			};
+			@include ${primaryMixinName}();
+		` : ''}
+	`;
+
+	// Build Sass
 	try {
-		const css = await buildSass(sassConfig);
+		const result = execa.sync(ftSass, [
```

Comment:
Same thing here, could we change this from a sync call to an async call as to not block the event-loop as other test sub-tasks could be running at this time as well